src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -261,11 +242,31 @@ public static double sensitivityAtConstantDepth(final int depth, final double[]
         final RouletteWheel qualityRW = new RouletteWheel(trimDistribution(qualityDistribution));
         final Random uniformRNG = new Random(randomSeed);
 
+        final Random r = new Random(randomSeed);
+
+        final BinomialDistribution bd = new BinomialDistribution(depth, alleleFraction);
+
+        double averageQuality = IntStream.range(0, qualityDistribution.length-1).mapToDouble(n -> n * qualityDistribution[n]).sum();
```

Comment:
add comment: "calculate mean and deviation of quality distribution to enable gaussian sampling below."